geomstats/invariant_metric.py

```diff
@@ -144,9 +144,7 @@ def left_exp_from_identity(self, tangent_vec):
         metric, it used the left-invariant metric associated to the same
         inner-product at the identity.
         """
-        from geomstats.spd_matrices_space import SPDMatricesSpace
-        spd_space = SPDMatricesSpace(n=self.group.dimension)
-
+        import geomstats.spd_matrices_space as spd_space
```

Comment:
mmmh that's not recommended by the style guide - you're supposed to keep the whole name.